src/controller/chat-bot-controller.ts

```diff
@@ -1,7 +1,7 @@
-import ChatBotService from '../service/chat-bot-service';
+import chatBotService, { ChatBotService } from '../service/chat-bot-service';
 
 class ChatBotController {
-  private chatBotService: ChatBotService = new ChatBotService();
+  private chatBotService: ChatBotService = chatBotService;
 
   chat = async ctx => {
     const { question } = ctx.request.body;

```

Comment:
The code patch seems to be a simple import change and a minor modification in the instantiation of the `ChatBotService` class. 

Improvement suggestions:
- It's good practice to use consistent naming conventions for variables, functions, and classes. In this case, the imported `chatBotService` variable uses camelCase while the `ChatBotService` class uses PascalCase. It would be better to use the same convention throughout the codebase.
- It's generally recommended to use dependency injection instead of instantiating dependencies within a class. This makes the code more modular and easier to test. Consider injecting the `ChatBotService` instance into the `ChatBotController` constructor instead of instantiating it within the class.